animations/staggered-animations/index.md

```diff
@@ -378,3 +378,9 @@ The following resources might help when writing animations:
 
 [Material motion spec](https://material.io/guidelines/motion/)
 : Describes motion for Material apps.
+
+## Other resources
+
+* For an alternate approach to sequence animation, see the
```

Comment:
I think we have to somehow make it clear that this is a community contributed package. 

Also, has any 'vetting' of this been made before we 'advertize' it?